src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -942,6 +942,12 @@ static void refda_adm_ietf_dtnma_agent_edd_var_list(refda_edd_prod_ctx_t *ctx)
      */
 }
 
+/* TODO:
+potential new CTRL's:
+ ensure-sbr, discard-sbr, ensure-tbr, discard-tbr
```

Comment:
Since rule object references include the object type, there can be a single `discard-rule` with a parameter union of `/aritype/sbr` and `/aritype/tbr` if that makes sense.
Along those same lines there can be `enable-rule` and `disable-rule` controls with the same parameter type.
In support of all three, there can be a TYPEDEF for a "rule" type that is the union of SBR and TBR references.

Reply:
Thanks, let me update the comment for now, then come back around to this after the execution code is in a better place, potentially as a follow-on PR.